### PR TITLE
Designed ForgotPassword Page

### PR DESCRIPTION
**Pull request by RajathJ**
Designed ForgotPassword Page #9

### CODE REVIEW

File: src/views/ForgotPassword.vue [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            memail:"",
+            error:true
+        }
+    },
+    methods:{
+        change:function(){
+            this.error=false;
+        },
+        protect_email:function(user_email){
+            if(user_email.length<=5){
```
Comment (rajankit911) [inline src/views/ForgotPassword.vue:49]: code formatting required.

File: src/views/ForgotPassword.vue [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                return "Invalid Email"
+            }
+            else{
+            var avg, splitted, part1, part2;
```
Comment (rajankit911) [inline src/views/ForgotPassword.vue:53]: use let or const instead of var

File: src/views/ForgotPassword.vue (hunk 1 above)
Comment (RajathJ) [reply src/views/ForgotPassword.vue:49]: Done with the indentation

File: src/views/ForgotPassword.vue (hunk 2 above)
Comment (RajathJ) [reply src/views/ForgotPassword.vue:53]: keyword changed

File: src/views/ForgotPassword.vue (hunk 1 above)
Comment (RajathJ) [reply src/views/ForgotPassword.vue:49]: Indentation Done

File: src/views/ForgotPassword.vue (hunk 2 above)
Comment (RajathJ) [reply src/views/ForgotPassword.vue]: keyword changed

**Pull request merged**